package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bellese/angular-design-system",
-  "version": "2.2.15",
+  "version": "2.2.18",
```

Comment:
@joshglazer should this be consistent with he version number from the package-lock file (also in this P/R). Also, what would cause this to jump from 15 to 18?